packages/repository-json-schema/src/build-schema.ts

```diff
@@ -35,10 +42,15 @@ export interface JsonSchemaOptions<T extends object> {
    */
   optional?: (keyof T)[];
 
+  /**
+   * Boolean to generate fieldNames maximizing compatibility (Azure)
```

Comment:
Can we make this more consistent with the others? E.g. `Set this flag to generate fieldNames maximizing compatibility (Azure)`? 

Also question, are `fieldNames` specifically for Azure?

Reply:
Description updated. The new format is less expressive but more compatible with all the possible formats in platforms apart from Azure. Checking the OpenAPI 3.0 spec, it should be fully compatible.
https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v3.0/schema.json#L196